### PR TITLE
bats: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/bats.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# bats
+
+> Bash Automated Testing System is a [TAP](https://testanything.org/)-compliant testing framework for Bash.
```
Comment (kbdharun) [inline pages/common/bats.md:3]:
```suggestion
> Bash Automated Testing System is a TAP-compliant testing framework for Bash.
```
Unfortunately, our clients don't support this rendering of links. Alternatively you can list it inside brackets.


```suggestion
> Bash Automated Testing System is a TAP (https://testanything.org/) compliant testing framework for Bash.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a minor suggestions for this page.

File: pages/common/bats.md (hunk 1 above)
Comment (kbdharun) [reply pages/common/bats.md]: > Why not in angled brackets "<>" as in the more information links?

Yeah, This should be enclosed inside an angle bracket. I forgot to add it while making the suggestion.

File: pages/common/bats.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# bats
+
+> Bash Automated Testing System is a TAP (https://testanything.org/) compliant testing framework for Bash.
```
Comment (kbdharun) [inline pages/common/bats.md:3]:
```suggestion
> Bash Automated Testing System is a TAP (<https://testanything.org/>) compliant testing framework for Bash.
```

Comment (vitorhcl) [reply pages/common/bats.md:3]:
```suggestion
> Bash Automated Testing System: a TAP (<https://testanything.org/>) compliant testing framework for Bash.
```
Just for consistency with other pages.

File: pages/common/bats.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/bats-core/bats-core>.
+
+- Run BATS tests in test script and output results in TAP (Test Anything Protocol) format:
```
Comment (vitorhcl) [inline pages/common/bats.md:6]:
```suggestion
- Run tests in a BATS test script and output results in the TAP (Test Anything Protocol) format:
```
I think this wording is less confusing.

Or even:
```suggestion
- Run a BATS test script and output results in the TAP (Test Anything Protocol) format:
```

File: pages/common/bats.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`bats --tap {{path/to/test.bats}}`
+
+- Count test cases without running any tests in the test script:
+
+`bats --count {{path/to/test.bats}}`
+
+- Run BATS tests in test scripts of a directory and subdirectories with file extension `.bats`:
```
Comment (vitorhcl) [inline pages/common/bats.md:14]:
```suggestion
- Run tests in BATS test scripts of a directory and subdirectories with file extension `.bats`:
```

File: pages/common/bats.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`bats --recursive {{path/to/directory}}`
+
+- Run BATS tests in test script and output results in desired format:
```
Comment (vitorhcl) [inline pages/common/bats.md:18]:
```suggestion
- Output results in a specific format:
```

File: pages/common/bats.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# bats
+
+> Bash Automated Testing System is a TAP (https://testanything.org/) compliant testing framework for Bash.
+> More information: <https://github.com/bats-core/bats-core>.
+
+- Run BATS tests in test script and output results in TAP (Test Anything Protocol) format:
+
+`bats --tap {{path/to/test.bats}}`
+
+- Count test cases without running any tests in the test script:
```
Comment (vitorhcl) [inline pages/common/bats.md:10]:
```suggestion
- Count test cases of a test script without running any tests:
```

File: pages/common/bats.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# bats
+
+> Bash Automated Testing System is a TAP (https://testanything.org/) compliant testing framework for Bash.
+> More information: <https://github.com/bats-core/bats-core>.
```
Comment (vitorhcl) [inline pages/common/bats.md:4]:
```suggestion
> More information: <https://bats-core.readthedocs.io/en/stable/usage.html>.
```
Always search for the project documentation. The priority is:
1. Official documentation
2. Official VCS (Version Control System, e.g. GitHub) link, if the README contain at least some examples
3. manned.org man page link

File: pages/common/bats.md (hunk 4 above)
Comment (SethFalco) [reply pages/common/bats.md:14]: I can't say I would be able to understand the original or the revised version of this command.

Maybe something more like:

```suggestion
- Run BATS test cases contained in a directory and subdirectories with file extension `.bats`:
```

**Review by gutjuri — APPROVED** [review]

**Review by SethFalco — APPROVED** [review]

File: pages/common/bats.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Run a BATS test script and output results in the TAP (Test Anything Protocol) format:
+
+`bats --tap {{path/to/test.bats}}`
+
+- Count test cases of a test script without running any tests:
+
+`bats --count {{path/to/test.bats}}`
+
+- Run BATS test cases contained in a directory and subdirectories with file extension `.bats`:
```
Comment (acuteenvy) [inline pages/common/bats.md:14]:
```suggestion
- Run BATS test cases contained in a directory and its subdirectories (files with a `.bats` extension):
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**